MaxText/kernels/megablox/common.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 The JAX Authors.
```

Comment:
Nit: update to `Copyright 2024 Google LLC`

Reply:
Resolved thanks Ran!